src/coreclr/jit/emitriscv64.cpp

```diff
@@ -2748,48 +2748,48 @@ ssize_t emitter::emitOutputInstrJumpDistance(const BYTE* src, const insGroup* ig
     return distVal;
 }
 
-static constexpr size_t NBitMask(uint8_t bits)
+static inline constexpr unsigned WordMask(uint8_t bits) noexcept
```

Comment:
Nit: `noexcept` is used nowhere in the JIT sources, so I wouldn't add it here either.